doc/documentation_data_format.rst

```diff
@@ -741,3 +741,118 @@ allows to specify multiple SBML models with corresponding condition and
+   "``,``", "8", "comma  as function argument separator", "left-to-right", "any", ""
+
+Note that operator precedence might not be as expected from other programming
+languages. Use parentheses to enforce the desired order of operations.
+
+Literals
+~~~~~~~~
+
+Numbers
++++++++
+
+There is no distinction between integer and floating point numbers. Numbers
+can be specified in decimal notation, e.g. ``1.0``, ``-1.0``, ``1.0e-3``,
+``1.0e3``.
+
+Booleans
+++++++++
+
+Boolean literals are ``true`` and ``false``. There is no automatic conversion
+between bool and float types. E.g., ``par * (par > 2)`` is not supported and
+should instead be implemented like ``piecewise(par, par > 2, 0)``.
+
+Functions
+~~~~~~~~~
+
+The following functions are supported:
+
+.. csv-table:: Supported functions
+   :header: "Function", "Argument types", "Comment", "Evaluates to"
+
+   "``pow(a, b)``", "float, float", "power function `b`-th power of `a`", "float"
+    "``exp(x)``", "float", "exponential function pow(e, x) (`e` itself not a supported symbol, but ``exp(1)`` can be used instead) ", "float"
+    "``sqrt(x)``", "float", "square root of x; ``pow(x, 0.5)``", "float"
+    "``log(a, b)``, ``log(x)``, ``ln(x)``, ``log2(x)``, ``log10(x)``", "float[, float]", "logarithm of ``a`` with base ``b``; ``log(x, e)``; ``log(x, e)``; ``log(x, 2)``; ``log(x, 10)`` (``log(0)`` is defined as ``-inf``) NOTE: ``log`` without explicit base is ``ln``, not ``log10``", "float"
+    "``sin``, ``cos``, ``tan``, ``cot``, ``sec``, ``csc``", "float", "trigonometric functions", "float"
+    "``arcsin``, ``arccos``, ``arctan``, ``arccot``, ``arcsec``, ``arccsc``", "float", "inverse trigonometric functions", "float"
+    "``sinh``, ``cosh``, ``tanh``, ``coth``, ``sech``, ``csch``", "float", "hyperbolic functions", "float"
+    "``arcsinh``, ``arccosh``, ``arctanh``, ``arccoth``, ``arcsech``, ``arccsch``,", "float", "inverse hyperbolic functions", "float"
+    "``piecewise(true_value_1, condition_1, [true_value_2, condition_2, ] [...] [true_value_n, condition_n,] false_value)``", "``true_value*`` and ``false_value`` are either all float or all bool, ``condition*`` are all bool", "The function value is ``true_value_1`` if ``condition_1`` is true, else ``true_value_2`` if ``condition_2`` is true, else ..., else ``true_value_n`` if ``condition_n`` is true, else ``false_value``.", "float"
+    "``abs(x)``", "float", "absolute value; ``piecewise(x, x>=0, -x)``", "float"
+    "``sign(x)``", "float", "sign of ``x``; ``piecewise(1, x>=0, -1)``", "float"
+    "``min(a, b)``, ``max(a, b)``", "float, float", "minimum / maximum of {a,b}; ``piecewise(a, a<=b, b)``, ``piecewise(a, a>=b, b)``", "float"
+
+Model time
+~~~~~~~~~~
+
+The model time is represented by the symbol ``time``. ``time`` always
+represents the current absolute simulation time (i.e., if the simulation starts
+at :math:`t_0 \neq 0`, then ``time`` is *not* the time since :math:`t_0`).
```

Comment:
```suggestion
The model time is represented by the symbol ``time``, which is the current simulated time, not the current duration of simulated time; if the simulation starts
at :math:`t_0 \neq 0`, then ``time`` is *not* the time since :math:`t_0`.
```